src/main/java/spoon/Metamodel.java

```diff
@@ -146,6 +149,9 @@ private Metamodel() { }
 		result.add(factory.Type().get(spoon.reflect.reference.CtUnboundVariableReference.class));
 		result.add(factory.Type().get(spoon.reflect.reference.CtVariableReference.class));
 		result.add(factory.Type().get(spoon.reflect.reference.CtWildcardReference.class));
+		result.add(factory.Type().get(CtImport.class));
+		result.add(factory.Type().get(CtImportKind.class));
+		result.add(factory.Type().get(CtWildcardStaticTypeMemberReference.class));
```

Comment:
remove interface but keep implementation and neat polymorphism

Reply:
Actually this is more complex than I was thinking as the whole metamodel (and associated tests) suppose that we have an interface for each implementation: if I want to put only implementation, I can't put any method to create it in factory etc.